pkg/cli/ui.go

```diff
@@ -1,13 +1,105 @@
-package clui
+package cli
 
 import (
 	"fmt"
 	"io"
 	"strconv"
+	"sync"
 
 	"github.com/fatih/color"
+	"github.com/izumin5210/clicontrib/pkg/clog"
+	"github.com/tcnksm/go-input"
 )
 
+type UI interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type UI should have comment or be unexported

Reply:
fixed in 3041c36